postgres/datadog_checks/postgres/metadata.py

```diff
@@ -0,0 +1,130 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import Dict, Optional, Tuple  # noqa: F401
+
+import time
+import json
+import psycopg2
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+from datadog_checks.base import is_affirmative
+from datadog_checks.base.utils.db.utils import (
+    DBMAsyncJob,
+    default_json_event_encoding,
+)
+from datadog_checks.postgres.connections import MultiDatabaseConnectionPool
+from datadog_checks.base.utils.tracking import tracked_method
+
+# default collection interval in minutes
+DEFAULT_COLLECTION_INTERVAL = 5
```

Comment:
Small python nit: adding a unit suffix `_ms` `_seconds` `_min` is appreciated if we're not standardizing on seconds (but I suggest aligning on seconds)